### PR TITLE
Create Hebrew Strings

### PR DESCRIPTION
**Pull request by Eng2Heb**

### CODE REVIEW

File: resources/language/Hebrew/strings.po [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+msgstr "אזור"
+
+# Script
+msgctxt "#32000"
+msgid "Warning"
+msgstr "אזהרה"
+
+msgctxt "#32001"
+msgid "Could not open stream for {0}!"
+msgstr "לא היתה אפשרות לפתוח את ההזרמה עבור {0}!"
```
Comment (beardypig) [inline resources/language/Hebrew/strings.po:37]: Is this a LTR rendering issue? is the `{0}` correctly rendered?

**Comment by beardypig** [comment]
Cool! :)

File: resources/language/Hebrew/strings.po (hunk 1 above)
Comment (Eng2Heb) [reply resources/language/Hebrew/strings.po:37]: Its the way github wysiwyg editor which is  LTR shows RTL text.
In kodi the text shows in correct way.
You can merge the strings.

Comment (beardypig) [reply resources/language/Hebrew/strings.po:37]: Cool.

**Pull request merged**